docs/visual-basic/developing-apps/programming/drives-directories-files/parsing-text-files-with-the-textfieldparser-object.md

```diff
@@ -35,10 +35,10 @@ translation.priority.ht:
   - "zh-cn"
   - "zh-tw"
 ---
-# Parsing Text Files with the TextFieldParser Object (Visual Basic)
+# Parsing text files with the TextFieldParser object (Visual Basic)
```

Comment:
There were two previous comments about eliminating "(Visual Basic)", but I'd leave it here unless you can think of a title that avoids having two uses of "with"